tests/testthat/helper-redact.R

```diff
@@ -17,3 +17,20 @@ redact_vetiver <- function(snapshot) {
 redact_port <- function(snapshot) {
     snapshot <- gsub(port, "<port>", snapshot, fixed = TRUE)
 }
+
+
+clean_python_tmp_dir <- function(variables) {
```

Comment:
@t-kalinowski Thank you so much for your suggestions on this!

Reply:
Glad it worked!